tests/foreman/api/test_capsulecontent.py

```diff
@@ -1037,16 +1025,18 @@ def test_positive_sync_collection_repo(
             '[galaxy_server.capsule_galaxy]\n'
             f'url={repo_path}\n'
         )
-        rhel7_contenthost.execute(f'echo "{cfg}" > ./ansible.cfg')
+
+        request.addfinalizer(lambda: target_sat.execute(f'rm -rf {cfg_path} {coll_path}'))
 
         # Try to install collections from the Capsule
-        result = rhel7_contenthost.execute(
+        target_sat.execute(f'echo "{cfg}" > {cfg_path}')
+        result = target_sat.execute(
             'ansible-galaxy collection install theforeman.foreman theforeman.operations'
```

Comment:
Just curious, if its okay to keep this collections installed on Satellite? if not then could we uninstall it in teardown?

Reply:
This would be cleaner for sure. I'd support this, ack otherwise

Reply:
I do the cleanup in finalizer (L1029) - I remove the collections installed in the custom `{coll_path}` and also remove the config created for the current user.